### ISSUE TITLE
xLights assigns shadow models to models randomly, why?

### ISSUE THREAD

**Issue by MrChristmas2000** [xLightsSequencer/xLights#3238]
I observed that after I removed all the shadow models from models that didn't need them xLights turns around and reassigns random models as shadow models. Is it supposed to do that and why?

**Comment by keithsw1111** [comment]
That sounds just like models end up overlapping because you removed something. I very much doubt it is a bug and if it is we are going to need a lot more information.

**Comment by MrChristmas2000** [comment]
I again deleted all the unwanted 'ShadowModelFor' code in the xlights_rgbeffects file.
I checked for overlapping models and there were none.
I went back an checked last years xlights_rgbeffects file and I indeed had deleted unwanted shadow models. I went from .40 to 2022.6 for this year and am now up to .9 and just noticed the return of the shadow models which pointed to an unused model which I have now deleted and will monitor if the shadow models show up again.
